### PR TITLE
should address failed shunit tests

### PR DESCRIPTION
**Pull request by Bajger**

### CODE REVIEW

File: src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st [hunk 1]
```diff
@@ -72,7 +72,7 @@ PhLImageCreateFromPullRequestCliCommand >> findTemplate [
 	"find template from Inria CI server based on Pull request number"
 
 	| imageFinder templateUrl |
-	imageFinder := PhLImageFinder fromInriaCIPullRequest: self pullRequestNumber.
+	imageFinder := self imageFinderFromPullRequest.
```
Comment (demarey) [inline src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st]: I would just define a methods #imageFinderClass on `PhLImageCreateFromPullRequestCliCommand` and `PhLImageCreateFromPullRequestCliCommandMock` with the corresponding class. The method call and the parameter are the same in both cases.

File: src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st [hunk 2]
```diff
@@ -82,6 +82,13 @@ PhLImageCreateFromPullRequestCliCommand >> findTemplate [
 		  url: templateUrl
 ]
 
+{ #category : #private }
+PhLImageCreateFromPullRequestCliCommand >> imageFinderFromPullRequest [
+	
+	"get image finder based on PR"
+	^  PhLImageFinder fromInriaCIPullRequest: self pullRequestNumber
+]
+
```
Comment (demarey) [inline src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st:90]: with previous comment, this method becomes useless

File: src/PharoLauncher-CLI-Tests/PhLImageCreateFromPullRequestCliCommandMock.class.st [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+{ #category : #accessing }
+PhLImageCreateFromPullRequestCliCommandMock >> imageName [
+
+	^ self implicitImageName 
+]
+
+{ #category : #private }
+PhLImageCreateFromPullRequestCliCommandMock >> outStream [
+	^ outStream
+		ifNil: [ outStream := VTermOutputDriver on: self context stdout ]
```
Comment (demarey) [inline src/PharoLauncher-CLI-Tests/PhLImageCreateFromPullRequestCliCommandMock.class.st]: I'm just wondering why it is needed?

**Review by demarey — APPROVED** [review]
Good work! Thanks David.

File: src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st (hunk 1 above)
Comment (Bajger) [reply src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st]: Done, using imageFinderClass instead.

File: src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st (hunk 2 above)
Comment (Bajger) [reply src/PharoLauncher-CLI/PhLImageCreateFromPullRequestCliCommand.class.st:90]: removed, using imageFinderClass instead.

File: src/PharoLauncher-CLI-Tests/PhLImageCreateFromPullRequestCliCommandMock.class.st (hunk 3 above)
Comment (Bajger) [reply src/PharoLauncher-CLI-Tests/PhLImageCreateFromPullRequestCliCommandMock.class.st]: You were right, I realized there is a way to set the CLI arguments without executing a command instance. Removed unnecessary methods.

**Pull request merged**